CHANGELOG.md

```diff
@@ -1,3 +1,63 @@
+## Fleet 4.40.0 (Nov 3, 2023)
+
+### Changes
+
+* **Endpoint management**:
```

Comment:
```suggestion
* **Endpoint operations**:
```

Maybe we call this endpoint ops? This way, the changelog matches what we're trying on fleetdm.com:
<img width="272" alt="Screenshot 2023-11-03 at 5 40 52 PM" src="https://github.com/fleetdm/fleet/assets/47070608/1df8fb97-031f-4086-859e-4e332177f6d4">

fleetdm.com/pricing:
<img width="289" alt="Screenshot 2023-11-03 at 5 41 13 PM" src="https://github.com/fleetdm/fleet/assets/47070608/0bce1241-5d4b-40b8-8c70-cac7dc69c42a">